test/bad_test_confs/nd96asr_v4.conf

```diff
@@ -50,7 +50,7 @@
 #  * || check_gpu_xid
 #  * || check_nvsmi_healthmon
  # modified expected value to fail
- * || check_cuda_bw 50
+ * || check_cuda_bw 50 250
```

Comment:
Is 'check_cuda_bw' meant to have two values? Or just one?

Reply:
This function can take one or both. The first is the GPU  dth/htd  expected BW and the second is GPU to GPU expected BW. This specific file is meant to produce a failure on dth/htd when unit testing. The second argument can be left off and it will still produce the failure. I added the second argument so that it looks like the default conf file for this SKU. Essentially, it's a formatting/cosmetic choice.